src/main/java/controller/planning/EventPlanningController.java

```diff
@@ -1,23 +1,64 @@
 package controller.planning;
 
+import db.DBConnectionProvider;
+import db.DBInitializer;
+import db.PostgresqlConnectionProvider;
+import model.User;
+import repository.UserRepository;
 import view.planning.EventPlanning;
 import view.planning.EventPlanningWindow;
 import view.planning.modelview.UserView;
 
+import java.sql.SQLException;
 import java.time.LocalDateTime;
 import java.util.List;
 
 public class EventPlanningController {
     private EventPlanning eventPlanningWindow;
+    private DBInitializer tablesInitializer;
+    private UserRepository userRepository;
 
     public void start() {
         eventPlanningWindow = new EventPlanningWindow();
         eventPlanningWindow.setOnSaveEvent(this::saveEventInDatabase);
+
+        DBConnectionProvider dbConnectionProviderForDBInitializer = new PostgresqlConnectionProvider();
+        try{
```

Comment:
These two tries should be merged. And it shouldn't `println` anything. There should be some kind of error handling, showing error dialog for example or exception logging

Reply:
Merged this two tries. Also added simple error dialog (JOptionPane.showMessageDialog)